ui/pages/confirmations/components/confirm/info/shared/transaction-details/transaction-details.tsx

```diff
@@ -83,6 +86,30 @@ export const MethodDataRow = () => {
   );
 };
 
+export const AmountRow = () => {
+  const t = useI18nContext();
+  const { currentConfirmation } = useConfirmContext<TransactionMeta>();
+  const { currency } = useUserPreferencedCurrency(PRIMARY);
+
+  const value = currentConfirmation?.txParams?.value;
+  const simulationData = currentConfirmation?.simulationData;
+
+  if (!value || value === '0x' || !simulationData?.error) {
```

Comment:
Should the condition include `(simulationData && !simulationData.error)` to ensure this row doesn't flicker while simulation is loading and only shows after simulation fails?